src/Kunstmaan/MediaBundle/Resources/views/Media/bulkUpload.html.twig

```diff
@@ -1,4 +1,43 @@
-{% extends '@KunstmaanMedia/Default/layout.html.twig' %}
+{% extends '@KunstmaanAdminList/Default/list.html.twig' %}
+
+
+{% block body %}
+    {% set lowestTopChild = foldermanager.getRootFolderFor(folder) %}
+    {% if lowestTopChild %}
+        {% set navigationChildren = foldermanager.getFolderHierarchy(lowestTopChild) %}
+    {% endif %}
+    {{ parent() }}
+{% endblock %}
+
+
+{% block sidebar %}
```

Comment:
Maybe we should add a new layout twig file for this. I can see that there are already 3 twig files with the sames blocks in it. Maybe put them in a twig file and extend from that template.